src/main/java/io/fabric8/vertx/maven/plugin/utils/PackageHelper.java

```diff
@@ -134,6 +146,8 @@ protected void generateManifest() {
             attributes.put(MAIN_VERTICLE, mainVerticle);
         }
 
+        ManifestUtils.addExtraManifestInfo(mavenProject, attributes);
```

Comment:
yeah I was thinking on that -- what if user had his own MANIFEST.MF .. lets discuss on it later